project/build.scala

```diff
@@ -7,9 +8,9 @@ object build extends Build {
   val pirate = Project(
     id = "pirate"
   , base = file(".")
-  , settings = Defaults.coreDefaultSettings ++ publishSettings ++ Seq[Sett](
+  , settings = Defaults.coreDefaultSettings ++ promulgate.library(s"com.ambiata.pirate", "ambiata-oss") ++ Seq[Sett](
       name := "pirate"
-    , organization := "io.mth"
+    , organization := "com.ambiata"
```

Comment:
@markhibberd Just talking to Huw - sounds like we can't make this Ambiata. I guess we should publish it as `io.mth.pirate`. Would that still go under `ambiata-oss`? Do we have another bucket?